app.js

```diff
@@ -9,9 +9,7 @@ Test this function by hand in the console to get it working, and when you think
 function sum(a,b){ //eslint-disable-line
   var sumMath = a + b;
   var arrayString = ('The sum of ' + a + ' and ' + b + ' is ' + sumMath + '.');
```

Comment:
variable naming is important, arrayString isn't descriptive of what this string stores.  A more descriptive name might be along the lines of sumTextOutput.